openpaas-james/mailbox/encrypted/api/src/main/scala/com/linagora/openpaas/encrypted/KeystoreManagerContract.scala

```diff
@@ -8,6 +8,7 @@ import reactor.core.scala.publisher.{SFlux, SMono}
 
 import scala.jdk.CollectionConverters._
 
+// Remark: why is this not in test folder? Need to move =>
```

Comment:
:+1: and we thus need to fix pom.xml as well...

Reply:
IMO it deserve a separate PR

Reply:
#113